src/fermilib/utils/_plane_wave_hamiltonian.py

```diff
@@ -26,6 +26,33 @@
 from projectq.ops import QubitOperator
 
 
+def wigner_seitz_length_scale(wigner_seitz_radius, n_particles, dimension):
+    """Function to give length_scale associated with Wigner-Seitz radius.
```

Comment:
You can leave it this way and we can address the issue of having a uniform convention when FermiLib gets ready for the first release.